test/functional/test_runner.py

```diff
@@ -646,6 +667,11 @@ def run_tests(*, test_list, src_dir, build_dir, tmpdir, jobs=1, enable_coverage=
                     logging.debug("Early exiting after test failure")
                     break
 
+                if "[Errno 28] No space left on device" in stdout:
+                    sys.exit(f"Early exiting after test failure due to insuffient free space in {tmpdir}\n"
++                            f"Test execution data left in {tmpdir}.\n"
++                            f"Additional storage is needed to execute testing.")
```

Comment:
In 858fa78637041ae704005d4b6e564dd8245f4b5d "test: Handle functional test disk-full error"

The `+`s at the beginning of the lines are unecessary.

```suggestion
                             f"Test execution data left in {tmpdir}.\n"
                             f"Additional storage is needed to execute testing.")
```

Reply:
fixed